tests/queries/0_stateless/03166_from_readable_size.sql

```diff
@@ -0,0 +1,50 @@
+-- Should be the inverse of formatReadableSize
```

Comment:
Could you add a comment at the top to please keep this test in-sync with 03166_from_decimal_readable_size.sql.

Reply:
Could you please rename the test and expected result files to 03166_fromReadableSize.sql? (basically named after the SQL function name?)

Reply:
Done!